src/main/java/mate/academy/bookstore/service/UserServiceImpl.java

```diff
@@ -21,6 +26,8 @@ public UserResponseDto register(UserRegistrationRequestDto requestDto)
             throw new RegistrationException("The user with this email is already exist "
                     + requestDto.getEmail());
         }
-        return userMapper.toDto(userRepository.save(userMapper.toModel(requestDto)));
+        User user = userMapper.toModel(requestDto);
+        user.setRoles(Set.of(roleRepository.findByName(Role.RoleName.USER)));

```

Comment:
Unnesessary call to DB. Why cant you just 
```suggestion
        user.setRoles(Set.of(Role.RoleName.USER));
```

Reply:
Because required type is Role, not the RoleName. And i can't create each time new object of Role because RoleName field is unique